src/nupic/proto/SpatialPoolerProto.capnp

```diff
@@ -26,7 +26,7 @@ struct SpatialPoolerProto {
   minPctOverlapDutyCycles @15 :Float32;
   minPctActiveDutyCycles @16 :Float32;
   dutyCyclePeriod @17 :UInt16;
-  maxBoost @18 :Float32;
+  boostStrength @18 :Float32;
```

Comment:
The only problem with this is that loading an old model would not cause an error. But I don't think that will be a problem. So this is fine with me.